<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Вход в систему</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(141, 196, 145);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="login_lbl">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Book Antiqua&quot;;</string>
    </property>
    <property name="text">
     <string>Логин</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="login_tb">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Book Antiqua&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="maxLength">
     <number>24</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="login_btn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
font: 14pt &quot;Book Antiqua&quot;;</string>
    </property>
    <property name="text">
     <string>Войти</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QLabel" name="pass_lbl">
    <property name="geometry">
     <rect>
      <x>73</x>
      <y>180</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Book Antiqua&quot;;</string>
    </property>
    <property name="text">
     <string>Пароль</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pass_tb">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="maxLength">
     <number>24</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="title_lbl">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Book Antiqua</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Войдите в систему</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>481</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <widget class="QPushButton" name="registr_btn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>310</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Book Antiqua&quot;;
text-decoration: underline;
color: rgb(0, 0, 0);
border: none;</string>
     </property>
     <property name="text">
      <string>Зарегистрироваться</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoRepeatDelay">
      <number>300</number>
     </property>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>login_tb</zorder>
   <zorder>login_btn</zorder>
   <zorder>pass_lbl</zorder>
   <zorder>pass_tb</zorder>
   <zorder>title_lbl</zorder>
   <zorder>login_lbl</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
